index.js

```diff
@@ -7,6 +7,7 @@ var express = require('express');
 var alexa = require('alexa-app');
 var verifier = require('alexa-verifier');
 var bodyParser = require('body-parser');
+var avm = require('alexa-verifier-middleware');
```

Comment:
I would call this `alexaVerifierMiddleware` to be consistent and remove the `var verifier` above that i think we no longer need?